src/main/java/com/cst438/controller/StudentController.java

```diff
@@ -124,6 +128,7 @@ public EnrollmentDTO addCourse(
         enrollment.setUser(student);
         enrollment.setSection(section);
         enrollmentRepository.save(enrollment);
+        gradebookServiceProxy.sendMessage("Enrollment created: " + enrollment.getEnrollmentId());
```

Comment:
should be "addEnrollment " and enrollment instead of enrollment.getEnrollmentId()